src/payload-types.ts

```diff
@@ -192,6 +193,7 @@ export interface Installation {
  */
 export interface PvProduction {
   id: number;
+  owner: number | User;
```

Comment:
can't we resolve the owner through the installation or is this not performant enough?